src/cosmosdb-preview/azext_cosmosdb_preview/_validators.py

```diff
@@ -170,6 +171,65 @@ def _parse_resource_path(resource,
     return result['resource_id']
 
 
+def validate_certificates(ns):
+    """ Extracts multiple comma-separated certificates """
+    if ns.external_gossip_certificates is not None:
+        ns.external_gossip_certificates = get_certificates(ns.external_gossip_certificates)
+    if ns.client_certificates is not None:
+        ns.client_certificates = get_certificates(ns.client_certificates)
+    if ns.gossip_certificates is not None:
+        ns.gossip_certificates = get_certificates(ns.gossip_certificates)
+
+
+def get_certificates(ns):
```

Comment:
I will suggest using another meaningful variable name instead of `ns`. `ns` is `namespace` carrying variables which will be passed to all validators by design.

Reply:
fixed.